tbx/project_styleguide/templates/patterns/atoms/motif-heading/motif-heading.html

```diff
@@ -1,5 +1,26 @@
-<h{{ heading_level|default:1 }} class="motif-heading {% if classes %}{{ classes }}{% endif %}" aria-label="{% firstof aria_label heading %}">
-    {% with first_letter=heading|slice:":1" %}
-        <span class="motif-heading__drop-cap {% if first_letter|lower == 'i' %}motif-heading__drop-cap--i{% endif %} {% if first_letter|lower in 'f,p' %}motif-heading__drop-cap--narrow{% endif %} {% if first_letter|lower == 'w' %}motif-heading__drop-cap--narrower{% endif %} {% if first_letter|lower in 't,y' %}motif-heading__drop-cap--narrowest{% endif %}">{{ first_letter }}</span><span>{{ heading|slice:"1:" }}</span>
-    {% endwith %}
-</h{{ heading_level|default:1 }}>
+{% load util_tags %}
+
+{% if heading_level|ifinlist:"2,3,4,5,6" %}
```

Comment:
The custom filter is necessary here as for a h1 tag, the `heading_level` value will either be `1` or an empty string (the default value when it is undefined), which is hard to check for (if you check `if heading_level|length > 0` it will evaluate to `True` for the other heading levels) - so we need to explicitly check if the `heading_level` is in the range 2 to 6.

Reply:
could this have been written as `{% if heading_level > 1 and heading_level < 7 %}` ?

Reply:
oh good suggestion will try that